templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -402,13 +355,17 @@ Parameters:
       - db.t2.large
       - db.t2.xlarge
       - db.t2.2xlarge
+      - db.t3.medium
+      - db.t3.large
+      - db.t3.xlarge
+      - db.t3.2xlarge
     ConstraintDescription: Must be a valid RDS instance class from the list
-    Description: RDS instance type
+    Description: RDS instance type (must be r4 family if using Aurora).
```

Comment:
I know this is consistent with our other templates but this can probably be `... r4, t3 or r5 if using Aurora`. We should update it for other products

Reply:
Right, I'll add the new families

Reply:
Disregard `t3`, it is not supported by the Aurora template. However `r5` is still a valid option

Reply:
✅